nautobot/docs/installation/upgrading-from-nautobot-v1.md

```diff
@@ -322,49 +323,59 @@ These endpoints `/ipam/roles/`, `/dcim/rack-roles/` and `/dcim/device-roles/` ar
 | SecretsGroupAssociation | `group`                   | `secrets_group`                  | `/extras/secrets-groups-associations/?secrets_group=<uuid/slug>`          |
-| Site                    | `tag`                     | `tags`                           | `/dcim/sites/?tags=<slug>`                                                |
-| Tenant                  | `tag`                     | `tags`                           | `/tenancy/tenants/?tags=<slug>`                                           |
+| Service                 | `tag`                     | `tags`                           | `/ipam/services/?tags=<uuid/slug>`                                        |
+| Site                    | `tag`                     | `tags`                           | `/dcim/sites/?tags=<uuid/slug>`                                           |
+| Tenant                  | `tag`                     | `tags`                           | `/tenancy/tenants/?tags=<uuid/slug>`                                      |
 | User                    | `changes`                 | `object_changes`                 | `/users/users/?object_changes=<id>`                                       |
 |                         | `has_changes`             | `has_object_changes`             | `/users/users/?has_object_changes=True/False`                             |
 |                         | `group`                   | `groups`                         | `/users/users/?groups=<slug>`                                             |
 |                         | `group_id`                | `groups_id`                      | `/users/users/?groups_id=<id>`                                            |
 | VirtualMachine          | `local_context_data`      | `local_config_context_data`      | `/virtualization/virtual-machines/?local_config_context_data=True/False`  |
 |                         | `local_context_schema_id` | `local_config_context_schema_id` | `/virtualization/virtual-machines/?local_config_context_schema_id=<uuid>` |
 |                         | `local_context_schema`    | `local_config_context_schema`    | `/virtualization/virtual-machines/?local_config_context_schema=<slug>`    |
+|                         | `tag`                     | `tags`                           | `/virtualization/virtual-machines/?tags=<uuid/slug>`                      |
+| VLAN                    | `tag`                     | `tags`                           | `/ipam/vlans/?tags=<uuid/slug>`                                           |
+| VMInterface             | `tag`                     | `tags`                           | `/virtualization/interfaces/?tags=<uuid/slug>`                            |
+| VRF                     | `tag`                     | `tags`                           | `/ipam/vrfs/?tags=<uuid/slug>`                                            |
 
 ### Enhanced Filter Fields
```

Comment:
Should we document the addition of filtering by `uuids` functionality in this section? It might be a little too much

Reply:
That'd be a set of additions to the following `Enhanced FilterFields` section wouldn't it? I was thinking that given `tags` is a "new" filter rather than an enhancement to the existing `tag` filter that it didn't belong there, but I'm open to convincing otherwise.

Reply:
I'm good with that